staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go

```diff
@@ -402,6 +406,20 @@ func TestConcurrentRequests(t *testing.T) {
 	waitGroup.Wait()
 }
 
+// Shows that versions within groups are ordered in the dicvoery document
+// by their assigned version priority
+func TestVersionOrdering(t *testing.T) {
```

Comment:
I think this is an incomplete version of the test `TestVersionSortingWithPriority`